### PR TITLE
HPCC-21708 Disk usage graphs added to ECL Watch doc

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review
@miguelvazq optionally review

See fully compiled PDF file for image rendering.
http://10.240.32.243/job/DocDev70/CE=ubuntu-18.04-amd64-1/70/artifact/build-docs/Release/docs/EN_US/The_ECL_Watch_Manual_EN_US-7.2.1-closedown0.pdf

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

## Testing:

Successful end to end unit test built at http://10.240.32.243/job/DocDev70/CE=ubuntu-18.04-amd64-1/

### CODE REVIEW

File: docs/EN_US/ECLWatch/TheECLWatchMan.xml [hunk 1]
```diff
@@ -685,6 +685,15 @@
       page in ECL Watch click on the <emphasis role="bold">ECL
       Watch</emphasis> image at the top of any page, as shown above.</para>
 
+      <sect2>
+        <title>Cluster Status Information Graphic</title>
+
+        <para>The Cluster Activity page displays an informative graphic bar
+        along the top where some graphs provide a glimpse of usage activity on
```
Comment (JamesDeFabia) [inline docs/EN_US/ECLWatch/TheECLWatchMan.xml]: The Cluster Activity page displays an informative graphic bar along the top where some graphs provide a glimpse of usage activity on the clusters.
s/b
The Cluster Activity page displays graphs along the top showing cluster disk usage.

File: docs/EN_US/ECLWatch/TheECLWatchMan.xml [hunk 2]
```diff
@@ -685,6 +685,15 @@
...
+        the clusters. Clicking on each of the images brings up more
+        information about the individual disk usage activity. </para>
```
Comment (JamesDeFabia) [inline docs/EN_US/ECLWatch/TheECLWatchMan.xml]: brings up s/b displays

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Two comments inline

**Comment by g-pan** [comment]
@JamesDeFabia please review revisions.

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV

**Comment by richardkchapman** [comment]
@miguelvazq Any comments?

**Comment by miguelvazq** [comment]
Looks good to me after revision.

**Pull request merged**